### ISSUE TITLE
EDITORIAL: Clarify what it means to support "ARIA 1.0 combobox"

### ISSUE THREAD

**Issue by zcorpan** [w3c/aria#1118]
> The ARIA 1.0 specification describes a combobox pattern where the input element with the combobox role references the popup element with aria-owns instead of aria-controls. User agents, assistive technologies, and conformance checkers SHOULD continue to support the ARIA 1.0 pattern so that existing implementations of the ARIA 1.0 pattern remain functional. The ARIA 1.1 specification describes a combobox pattern where the element with role combobox is a composite container instead of a focusable input. User agents and assistive technologies did not adequately support the ARIA 1.1 pattern, so the ARIA 1.2 pattern supersedes the ARIA 1.1 pattern, which authors SHOULD abandon.

http://w3c.github.io/aria/#combobox

* Why should conformance checkers support the ARIA 1.0 pattern? If we want authors to use the 1.2 pattern, shouldn't conformance checkers flag an error when the 1.0 pattern is used?
* What does it mean for UAs and ATs to support the ARIA 1.0 pattern? If it is required for web compat to support the legacy markup pattern, the processing model to handle both the 1.2 pattern and the 1.0 pattern should be clearly specified in the current spec.

**Comment by accdc** [comment]
I agree this text is wrong.

We are going to have to address the issue where markup validation software should be flagging the 1.0 design pattern when aria-owns is detected, and then state that aria-controls needs to be used instead.

This simple fix makes all such comboboxes immediately work correctly across all mainstream platforms and ATs that currently support ARIA.

**Comment by mcking65** [comment]
@zcorpan, would it be OK with you to mark this a duplicate of #1178? We are discussing the same text in #1178, and I have proposed a revision there. Actually, technically 1178 is the dupe as this came first, but I was working backward in time.

**Comment by jnurthen** [comment]
Close as duplicate of #1178 - please reopen if disagree
